tests/unit/event/event_core.js

```diff
@@ -25,12 +24,9 @@ QUnit.module( libName, {
 		// the collections existence in non touch enabled test browsers
 		$.Event.prototype.touches = [ { pageX: 1, pageY: 1 } ];
 
-		$.mobile.pageContainer = originalPageContainer || $( "body" );
-
-		$( $.mobile.pageContainer ).unbind( "throttledresize" );
+		$( ".ui-pagecontainer" ).unbind( "throttledresize" );
```

Comment:
this should actually be `$( "body" )` `pagecontainer` is not even included in this test this is only passing because unbind will not throw on an empty collection